backend/app/services/channels/messager_config.py

```
@@ -0,0 +1,65 @@
+
+def _get_channel_spec(channel_id: int) -> Optional[Dict[str, Any]]:
+    """Load the active Messager CRD spec for a channel."""
+    db = SessionLocal()
+    try:
+        channel = (
+            db.query(Kind)
+            .filter(
+                Kind.id == channel_id,
+                Kind.kind == MESSAGER_KIND,
+                Kind.user_id == MESSAGER_USER_ID,
+                Kind.is_active == True,
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)